### PR TITLE
Adds a tour step for the contextual menu on the graph

### PR DESCRIPTION
**Pull request by cfcosta**
Closes: https://github.com/kiali/kiali/issues/1750

Kiali_Console

### CODE REVIEW

**Comment by cfcosta** [comment]
@mtho11 is this what you had in mind?

File: src/pages/Graph/GraphHelpTour.ts [hunk 1]
```diff
@@ -19,7 +19,14 @@ export const GraphTourStops: { [name: string]: TourStopInfo } = {
...
     description:
-      "Single click a node to see its summary and emphasize its end-to-end paths.  Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
+      "Single click a node to see its summary and emphasize its end-to-end paths. Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
+    position: PopoverPosition.auto,
+    offset: '0, -350'
+  },
+  ContextualMenu: {
+    name: 'Contextual Menu',
+    description:
+      'Right click a node to see the contextual menu for the node, such as details, traffic and inbound/outbound metrics.',
```
Comment (mtho11) [inline src/pages/Graph/GraphHelpTour.ts]: You can also right-click on an edge. The description should reflect that option as well.

Comment (cfcosta) [reply src/pages/Graph/GraphHelpTour.ts]: @mtho11 done, also changed the weird structure of the phrase for something better.

Comment (cfcosta) [reply src/pages/Graph/GraphHelpTour.ts]: Just realized I forgot to add a screenshot, thanks!

File: src/pages/Graph/GraphHelpTour.ts [hunk 2]
```diff
@@ -19,7 +19,14 @@ export const GraphTourStops: { [name: string]: TourStopInfo } = {
   Graph: {
     name: 'Graph',
     description:
-      "Single click a node to see its summary and emphasize its end-to-end paths.  Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
+      "Single click a node to see its summary and emphasize its end-to-end paths. Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
```
Comment (mtho11) [inline src/pages/Graph/GraphHelpTour.ts]: Change 'Single click' to 'Single-Click'

File: src/pages/Graph/GraphHelpTour.ts [hunk 3]
```diff
@@ -19,7 +19,14 @@ export const GraphTourStops: { [name: string]: TourStopInfo } = {
...
+    position: PopoverPosition.auto,
+    offset: '0, -350'
+  },
+  ContextualMenu: {
+    name: 'Contextual Menu',
+    description:
+      'Right click a node or an edge to see the contextual menu with links to details, traffic and inbound/outbound metrics for the node or edge.',
```
Comment (mtho11) [inline src/pages/Graph/GraphHelpTour.ts]: Change 'Right click' to 'Right-click'.  Just more proper English.

File: src/pages/Graph/GraphHelpTour.ts (hunk 2 above)
Comment (cfcosta) [reply src/pages/Graph/GraphHelpTour.ts]: Done!

File: src/pages/Graph/GraphHelpTour.ts (hunk 3 above)
Comment (cfcosta) [reply src/pages/Graph/GraphHelpTour.ts]: Done!

**Pull request merged**